meinberlin/assets/scss/components/_item_detail_2.scss

```diff
@@ -1,7 +1,7 @@
 .item-detail-2__smalltitle {
     font-weight: bold;
     font-size: inherit;
-    margin: 1em 0 0;
+    margin-top: 0;
```

Comment:
@khamui is this class used elsewhere? if so have you checked that changing it doesn't break styling?

Reply:
yes it is used, but those are fixed (and improved) by this line

https://github.com/liqd/a4-meinberlin/blob/21bfccd629aca0e42209de00535fef11f2742b25/meinberlin/assets/scss/components/_item_detail_2.scss#L55

Reply:
what about when there isn't the info-box?

Reply:
hmm..good point! yes thats not working for this case.

Reply:
I restored the margin-top for smalltitles which are not inside of the box. also good for mobile, if they get wrapped the still have spacing inbetween